eldritch/events.py

```diff
@@ -2480,13 +2505,36 @@ def PassFail(character, condition, pass_result, fail_result):
   return Sequence([condition, outcome], character)
 
 
-class Arrested(Sequence):
+class Arrested(Event):
 
   def __init__(self, character):
-    super().__init__([
-        ForceMovement(character, "Police"), LoseTurn(character),
-        Loss(character, {"dollars": values.Calculation(character, "dollars", operator.floordiv, 2)})
-    ], character)
+    super().__init__()
+    self.character = character
+    self.jail = None
+    self.done = False
+
+  def resolve(self, state):
+    if self.jail is None:
```

Comment:
The logging is clearer if it's its own event—otherwise (even with an overridden `log()` method) you get something like 
```
     Dummy's movement
       Dummy failed a will +0 check
         Dummy rolled 3
       Dummy moved to Police
       Dummy lost their next turn
```